test/cnudie/cnudie_util_test.py

```diff
@@ -338,3 +393,141 @@ def test_diff_label():
     with pytest.raises(RuntimeError) as re:
         label_diff = cnudie.util.diff_labels(left_labels=left_labels, right_labels=right_labels)
     assert re != None
+
+
+def test_determine_component_name():
+    component_name = cnudie.util.determine_component_name(
+        repository_hostname='example.com',
+        repository_path='comp_a',
+    )
+    assert component_name == 'example.com/comp_a'
+
+    component_name = cnudie.util.determine_component_name(
+        repository_hostname='example.com',
+        repository_path='Comp_A',
+    )
+    assert component_name == 'example.com/comp_a'
+
+
+class DetermineMainComponentTest(unittest.TestCase):
```

Comment:
I would prefer pytest-based tests; tmpdir might be injected via fixtures (https://docs.pytest.org/en/stable/fixture.html)

.. we are doing python, and not Java :-)
(I know, there is a legacy-base of junit-style tests in cc-utils. however, I migrate those to pytest, and write new tests excusively based on pytest..)